experiment_prototype/experiment_prototype.py

```diff
@@ -1616,13 +1616,15 @@ def check_slice(self, exp_slice):
 
                 # Check if any scanbound times are shorter than the intt.
                 if len(exp_slice['scanbound']) == 1:
-                    if exp_slice['scanbound'][0] * 1000 < exp_slice['intt']:
+                    tolerance = math.isclose(exp_slice['scanbound'][0] * 1000, exp_slice['intt'])
```

Comment:
this will also result in an error if intt is much lower than scanbound (enough to be out of the relative tolerance default of math.isclose, 1e-9. Although not the most efficient, I think that'd be fine to run, so I'd rather keep the original < condition here, but maybe define the tolerance as 1e-9 and ensure that scanbound+tolerance < intt.

Reply:
sorry, I guess it'd be throw an error if scanbound < intt-tolerance ?

Reply:
The reason this was added was that I was finding that these calculations have some slight floating point error and are inexact. I could possibly change it back if I round the scan bound calculations?

Reply:
I just think the functionality has changed. I think it now will throw an error even when intt << scanbound[0], which I don't think is an error condition (would just cause the program to wait for the boundary). Rounding or adding tolerance value to the < expression might solve the issue and maintain the original function?